README.md

```diff
@@ -81,9 +99,23 @@ Several parameters can be specified using environment variables:
 | `zookeeper_watcher_status`        | `gauge`   | Status of watcher: -1 = NOT_STARTED, 0 = OK, 1 = WARNING, 2 = ERROR |
 | `zookeeper_write_failure_count`   | `counter` | Number of failures writing messages               |
 
+### InfluxDB Watcher
+
+| Metric                            | Type      | Description                                       |
+|-----------------------------------|-----------|---------------------------------------------------|
+| `influxdb_average_round_trip_time` | `gauge` | Average Round Trip Time in seconds |
+| `influxdb_dropped_message_count` | `counter` | Number of messages that were dropped              |
+| `influxdb_max_round_trip_time`   | `gauge`   | Maximum Round Trip Time in seconds                |
+| `influxdb_min_round_trip_time`   | `gauge`   | Minimum Round Trip Time in seconds                |
+| `influxdb_read_failure_count`    | `counter` | Number of failures reading messages               |
+| `influxdb_running_average_round_trip_time` | `gauge` | Running Average Round Trip Time in seconds for last 5 messages |
+| `influxdb_watcher_status`        | `gauge`   | Status of watcher: -1 = NOT_STARTED, 0 = OK, 1 = WARNING, 2 = ERROR |
```

Comment:
does this represent influxdb being down?

Reply:
Yes, this indicates the health of the InfluxDB that the watcher is watching. @craigbr What do you think about a name change? Maybe take out the watcher and just do influxdb_status and kafka_status etc...